client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1918,3 +1931,23 @@ def get_errors(self):
 
     def create_failed(self, creator_data):
         self._failed_created_publish_instances.append(creator_data)
+
+
+def discover_template_placeholder_plugins(*args, **kwargs):
+    return discover(PlaceholderPlugin, *args, **kwargs)
+
+
+def register_template_placeholder_plugin(plugin: PlaceholderPlugin):
+    register_plugin(PlaceholderPlugin, plugin)
+
+
+def deregister_template_placeholder_plugin(plugin: PlaceholderPlugin):
+    deregister_plugin(PlaceholderPlugin, plugin)
+
+
+def register_template_placeholder_plugin_path(path: str):
+    register_plugin_path(PlaceholderPlugin, path)
+
+
+def deregister_template_placeholder_plugin_path(path: str):
+    deregister_plugin_path(PlaceholderPlugin, path)
```

Comment:
@iLLiCiTiT Would you like me to refactor these as well? `template_placeholder_plugin` to `workfile_build_plugin`?

Reply:
Done with https://github.com/ynput/ayon-core/pull/425/commits/90ece1cdf98cabeec76cf52bc7854667818f8d3f